Gems/Blast/Code/Source/Editor/EditorBlastFamilyComponent.cpp

```diff
@@ -133,4 +134,12 @@ namespace Blast
     {
         return AZ::Interface<AzPhysics::SystemInterface>::Get()->GetConfiguration()->m_materialLibraryAsset.GetId();
     }
+
+    AZ::Data::AssetId EditorBlastFamilyComponent::GetDefaultBlastAssetId() const
+    {
+        // Used for Edit Context.
+        // When the blast material asset property doesn't have an asset assigned it
+        // will show "(default)" to indicate that the default material will be used.
+        return {};
```

Comment:
Just be clear, does this mean that there is a default material that is part of the system and does not require an asset?

Reply:
Correct. Blast materials can be created from a MaterialConfiguration directly, no need for having a MaterialAsset. So when the editor component detects there is no material asset in the property it creates a material using the default values of MaterialConfiguration.